packages/playground/src/domains/agent-builder/components/agent-edit/agent-profile/__tests__/ready.test.tsx

```
@@ -0,0 +1,124 @@
+// @vitest-environment jsdom
+import { cleanup, fireEvent, waitFor } from '@testing-library/react';
+import { useLocation } from 'react-router';
```

ESLint

[error] '/home/jailuser/git/packages/playground/node_modules/react-router/dist/development/index.mjs' imported multiple times.

(import/no-duplicates)